### PR TITLE
add list of issues with comments

### PR DESCRIPTION
**Pull request by jeanheck**

### CODE REVIEW

**Comment by Wuerike** [comment]
Teste

File: app/server/resources.py [hunk 1]
```diff
@@ -20,8 +20,10 @@ def get_most_popular_repo(user: str):
 def get_issue_with_most_comments(user: str, repository: str):
   url = '/repos/{user}/{repository}/issues?state=open'.format(user=user, repository=repository)
 
-  url2 = '/repos/{user}/{repository}/issues/comments'
+  url2 = '/repos/{user}/{repository}/issues/comments'.format(user=user, repository=repository)
```
Comment (Wuerike) [inline app/server/resources.py:23]: fdsfdsf

**Review by Wuerike — APPROVED** [review]

**Comment by jeanheck** [comment]
asdasd

**Comment by jeanheck** [comment]
dasdasd

**Pull request closed without merging**